### PR TITLE
 component 'filactu' created, with min CSS

### PR DESCRIPTION
**Pull request by Olif51**

### CODE REVIEW

File: frontend/src/components/FilActu.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+function FilActu() {
+  return (
+    <div>
+      <h1>CHSCT</h1>
+      <h3>
```
Comment (rocambille) [inline frontend/src/components/FilActu.jsx:5]: looks more like a p

File: package.json [hunk 2]
```diff
@@ -17,6 +17,10 @@
   "license": "ISC",
   "devDependencies": {
     "concurrently": "^7.0.0",
-    "husky": "^7.0.0"
+    "husky": "^7.0.0",
+    "prettier": "2.8.0"
+  },
+  "dependencies": {
+    "eslint": "^8.29.0"
```
Comment (rocambille) [inline package.json:24]: no need for this : restore the file :)

File: frontend/src/App.jsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-
 import "./App.css";
+import FilActu from "./components/FilActu";
 
 function App() {
   return (
     <div className="App">
-      <Home />
+
+      <FilActu />
```
Comment (rocambille) [inline frontend/src/App.jsx:8]: in english please, and without shorthand

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
https://github.com/WildCodeSchool/2022-09-JS-Reims-project-3-enedis/actions/runs/3628956922/jobs/6120593275

File: frontend/src/App.jsx (hunk 3 above)
Comment (Olif51) [reply frontend/src/App.jsx:8]: Done, replace with "NewsFeed"

File: frontend/src/components/FilActu.jsx (hunk 1 above)
Comment (Olif51) [reply frontend/src/components/FilActu.jsx]: replace div by p

File: package.json (hunk 2 above)
Comment (Olif51) [reply package.json:24]: done ( I think)

**Pull request closed without merging**